### PR TITLE
A fix to processing entries for zenodo, print ORCID URL in interactive mode, perform some internal code refactorings

### PR DESCRIPTION
**Pull request by yarikoptic**
One commit addresses #77 (has proper Closes).  Individual commits have description on the changes so best to review one commit at a time.

### CODE REVIEW

File: tributors/main/orcid.py [hunk 1]
```diff
@@ -129,7 +130,7 @@ def get_orcid_token():
     return orcid_token
 
 
-def record_search(url, email, interactive=False):
+def record_search(url, email, interactive=False, how=""):
```
Comment (vsoch) [inline tributors/main/orcid.py]: I can tell what "how" is based on reading below how the functions are using it - can we use a more descriptive label here, like search type, and document this in the docstring so it's clear the string is just for informational purposes and doesn't derive any logic?

Comment (yarikoptic) [reply tributors/main/orcid.py]: done in c76a90ad4e9ae51c9b4b07fc3ca0df62b1f2d147

**Review by vsoch — COMMENTED** [review]
If you’ve tested this and are happy, LGTM. Last final tweak is to please bump the version and update the CHANGELOG.md with a description of your fixes. Thank you!

**Comment by yarikoptic** [comment]
> Last final tweak is to please bump the version and update the CHANGELOG.md with a description of your fixes.

I have merged #74, so hopefully it all would just get released when we merge it. I added `minor` label, so it should make it `0.1.0`.

**Comment by vsoch** [comment]
Hmm that approach seems super complex compared to what I usually do, but if you are managing it I’m indifferent. LGTM merge and release as you like.

**Comment by yarikoptic** [comment]
> Hmm that approach seems super complex compared to what I usually do, but if you are managing it I’m indifferent. LGTM merge and release as you like.

Initial setup is indeed a bit "complex" but whenever it works - it can't be simpler, just
- use PR descriptions worth a changelog
- label PRs with labels `minor`, `patch`, `documentation`, `internal` respectively
- add `release` label to the PR after which you would like to have release done.  So I am adding `release` label now and merging -- let's see if setup works or needs a few more tweeks

**Pull request merged**